cookie.cabal

```diff
@@ -18,7 +18,7 @@ library
                    , bytestring                >= 0.9.1.4
                    , blaze-builder             >= 0.2.1
                    , old-locale                >= 1
-                   , time                      >= 1.4
+                   , time                      >= 1.5
```

Comment:
parseTimeM was added in 1.5 https://github.com/haskell/time/commit/1a633e6737ae45634619e2c9895c89848020bdec